usermods/TTGO-T-Display/platformio_override.ini

```diff
@@ -1,8 +1,49 @@
-[env:esp32dev]
-build_flags = ${common.build_flags_esp32} 
-; PIN defines - uncomment and change, if needed:
-;  -D LEDPIN=2
-   -D BTNPIN=35
+# Example PlatformIO Project Configuration Override
+# ------------------------------------------------------------------------------
+# Copy to platformio_override.ini to activate overrides
+# ------------------------------------------------------------------------------
+# Please visit documentation: https://docs.platformio.org/page/projectconf.html
+
+[platformio]
+#default_envs = WLED_tasmota_1M  # define as many as you need
+default_envs = WLED_T-Display  # define as many as you need
+#----------
+# SAMPLE
+#----------
+[env:WLED_T-Display]
+
+board = esp32dev
+upload_speed = 921600
+monitor_speed = 115200
+platform = ${esp32.platform}
+platform_packages = ${esp32.platform_packages}
+build_unflags = ${common.build_unflags}
+build_flags = ${common.build_flags} ${esp32.build_flags} -D WLED_RELEASE_NAME=\"ESP32\" 
```

Comment:
Should be something like WLED_T-Display for the release name to indicate this is not a vanilla ESP32 build

Reply:
Just fixed this - thanks!